bundles/automation/org.eclipse.smarthome.automation.module.script.rulesupport/src/main/java/org/eclipse/smarthome/automation/module/script/rulesupport/shared/simple/SimpleRule.java

```diff
@@ -181,7 +181,7 @@ public void setConfigurationDescriptions(@Nullable List<ConfigDescriptionParamet
 
     @Override
     public List<Condition> getConditions() {
-        return conditions;
+        return conditions == null ? new ArrayList<>() : conditions;
```

Comment:
Is there a reason to create an `ArrayList` here instead of using `Collections.emptyList()` ?

Reply:
That was in the code before like this, but I agree that empty list would be the better choice.